### PR TITLE
Upgrade PHPUnit from 9 to 10

### PR DESCRIPTION
**Pull request by Shadow243**
Related Task: [https://avan.tech/item106800](https://avan.tech/item106800)

### CODE REVIEW

**Comment by marclaporte** [comment]
Please continue working on this, but wait a bit before merging in.

I'd like https://hub.docker.com/r/cypht/cypht to be operational ASAP.

**Comment by Shadow243** [comment]
> Please continue working on this, but wait a bit before merging in.
> 
> I'd like https://hub.docker.com/r/cypht/cypht to be operational ASAP.

This one is ready to be reviewed.

File: .travis/run.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ selenium_tests() {
 
 BUILD="$DB$TRAVIS_PHP_VERSION"
 case "$BUILD" in
-    postgresql7.4)
+    postgresql8.1)
```
Comment (josaphatim) [inline .travis/run.sh:13]: Not sure here. @kroky can you please enlighten us here. Since I see that the latest release of postgres is 16.3

Comment (kroky) [reply .travis/run.sh:13]: This is php version appended to the kind of db variable, so it ends up as postgresql (kind of db) and then the php version - 8.1 is correct here.

**Comment by Shadow243** [comment]
I'm separating tests to prevent the non-execution of some, like here:
```
╰─ php vendor/bin/phpunit --bootstrap vendor/autoload.php --configuration tests/phpunit/phpunit.xml --testdox --filter=Output_Test   
PHPUnit 10.5.20 by Sebastian Bergmann and contributors.

Runtime:       PHP 8.1.28
Configuration: /Users/shadow243/software/www/TIKI/cypht/tests/phpunit/phpunit.xml

No tests executed!
```

**Comment by Shadow243** [comment]
I'm separating tests to prevent the non-execution of some, like here:
```
 php vendor/bin/phpunit --bootstrap vendor/autoload.php --configuration tests/phpunit/phpunit.xml --testdox --filter=Hm_Test_Modules_Output
PHPUnit 10.5.20 by Sebastian Bergmann and contributors.

Runtime:       PHP 8.1.28
Configuration: /Users/shadow243/software/www/TIKI/cypht/tests/phpunit/phpunit.xml

No tests executed!
```

**Comment by Shadow243** [comment]
> I'm separating tests to prevent the non-execution of some, like here:
> 
> ```
>  php vendor/bin/phpunit --bootstrap vendor/autoload.php --configuration tests/phpunit/phpunit.xml --testdox --filter=Hm_Test_Modules_Output
> PHPUnit 10.5.20 by Sebastian Bergmann and contributors.
> 
> Runtime:       PHP 8.1.28
> Configuration: /Users/shadow243/software/www/TIKI/cypht/tests/phpunit/phpunit.xml
> 
> No tests executed!
> ```

Done.

**Comment by josaphatim** [comment]
@Shadow243 Look at the picture below. This is the output I'm getting. Can you just fix **The configuration file did not pass validation!** warning and make sure **user unittestuser already exists** pass correctly please ? Thanks.

**Comment by Shadow243** [comment]
You still have phpunit 9.x you should have 10.5.x instead. Checkout the picture.

**Comment by josaphatim** [comment]
You are right! But I'm still getting almost same thing. Instead I realize that there are now 296 tests instead of 379 and 710 assertions instead of 852.

Screenshot 2024-06-03 at 05 04 21

**Comment by Shadow243** [comment]
> assertions
Please compare with this list and not the number of tests and assertions.
Screenshot 2024-06-03 at 07 33 25

**Comment by Shadow243** [comment]
> Rename phpunit/modules/core/modules.php to make other tests availlable

Other tests renamed: https://github.com/cypht-org/cypht/pull/1044/commits/5825aa13de45b8b47219739d6b77e48da3f24edc

**Comment by josaphatim** [comment]
@kroky now looks good to me. Can you check please ?

**Pull request merged**